cwatm/hydrological_modules/water_quality/erosed.py

```diff
@@ -39,15 +40,107 @@ class waterquality_erosed(object):
     def __init__(self, model):
         self.var = model.var
         self.model = model
+        self.waterquality_vars = waterquality_vars(model)
 
     def initial(self):
         """
                 INITIAL PART OF THE EROSED MODULE
+
+        Sediment yield per grid cell is calculated with the Modified Universal Soil Loss Equation (MUSLE)
+        Williams (1995)
         """
-        print('erosed init')
+        self.waterquality_vars.initial()
+        # load initial MUSLE maps
+        # exponential function for fraction of rock (exp-0.053 * fcr)
+        self.var.fcfr = 1
+
+        # K_usle: USLE soil erodibility factor
+        self.var.kFactor = self.var.Kf = loadmap('kFactor')
+
+        # C_usle: USLE cover and management factor
+        self.var.cFactor = loadmap('cFactor')
+
+        # ls_usle: USLE topographic factor (slope-length)
+        self.var.lsFactor = loadmap('lsFactor')
+
+        # manning overland rougness: values for landcoverclasses from settingsfile
+        overlandManningVars = ['manForest', 'manGrassland', 'manirrPaddy', 'manirrNonPaddy']
+        self.var.manOverland = np.tile(globals.inZero, (4, 1))
+        i = 0
+        for variable in overlandManningVars:
+            self.var.manOverland[i] += loadmap(variable)
+            i += 1
+        # manningsN channel
+        self.var.manNChan = loadmap('chanMan')
+        # grid slope
+        tanslope = loadmap('tanslope')
+
+        # setting slope >= 0.00001 to prevent 0 value
+        self.var.tanslope = np.maximum(tanslope, 0.00001)
+
+        # tov: time of overland ocncentration
+        tov = divideArrays(np.power(self.var.lsFactor, 0.6) * np.power(self.var.manOverland, 0.6), 18 * np.power(self.var.tanslope, 0.3))
+
+        # channel flow time of concentration
+        #tch = divideArrays(0.62 * self.var.chanLength * np.power(self.var.manNChan, 0.75), np.power(self.var.cellArea, 0.125) * np.power(self.var.chanGrad, 0.375))
+
+        #print('tch', np.nanmean(tch)
+        # tconc: time of concentration for the grid
+            # provides very high values:  axis = 0: [1930.83034355  363.97959098 1952.27786728 ... 1685.05161614 3080.56903285
+                                         #  2302.08594123]
+                                        # axis = 1: [3875.37793006 3875.34839299 3875.3501527  3875.3501527 ]
+
+        #print('tov:', np.nanmean(tov, axis=1))
+        #print('tch_min:', np.nanmin(tch), 'tch_max:', np.nanmax(tch), 'tch_mean', np.nanmean(tch))
+        #print('tconc:', np.nanmean(self.var.tconc, axis=1))
+        print('chanman min:', np.nanmin(self.var.chanMan), 'max:', np.nanmax(self.var.chanMan), 'mean:', np.nanmean(self.var.chanMan))
```

Comment:
If it is a working version you can start cleaning and removing the 'print' lines, or at least comment it out

Reply:
I use those lines to troubleshoot and to see what happens, if I make changes. 
Maybe working version was an overstatement.